guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -7,5 +7,12 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Google Compute resouce as plugin
+
+Google compute resource have been extracted to the [Foreman Google](https://github.com/theforeman/foreman_google) plugin.
+
+Note: Foreman instances with enabled Google compute resource will be migrated automatically to the plugin.
+On new Foreman instances, Google compute resource first have to be enabled by running `foreman-installer --enable-foreman-plugin-google --enable-foreman-cli-google`
```

Comment:
Also made a wording suggestion.
```suggestion
Note: Foreman instances with enabled Google compute resource are migrated automatically to the plugin.
On new Foreman instances, you must first enable Google compute resource by running `foreman-installer --enable-foreman-plugin-google --enable-foreman-cli-google`
```

Reply:
I'd leave out the instructions for new instances and just stick to a note for the existing users of `foreman-gce`.